bundles/ch.elexis.core.l10n/src/ch/elexis/core/l10n/messages.properties

```diff
@@ -8618,6 +8622,10 @@ SystemPreferencePage_editor_labelText_1 = station identification id
 
 SystemPreferencePage_editor_selfContact_label = Hauptkontakt
 
+Template_not_editable = Umbenennen nicht m�glich
```

Comment:
Die Umlaute bitte generell in UTF-8 hex Schreibweise z.B. \u00F6 für ö \u00E für ä etc.